middleware/logRequest.js

```diff
@@ -1,10 +1,20 @@
+const { validateEmployeeData } = require("./validateEmployeeData");
+
 function logRequest(req, res, next) {
     console.log(`\nRequest Method: ${req.method} \t Request URL: ${req.url}`);
     if (req.method === "PATCH" || req.method === "POST") {
         console.log(`Request Body:`);
         console.log(req.body);
     }
+    if (req.url === '/employees/add') {
```

Comment:
I think this validate should also be applicable for update too

Reply:
Okay Ram